radio/src/targets/common/arm/stm32/stm32_spi.h

```diff
@@ -36,6 +36,7 @@ struct stm32_spi_t {
   uint32_t       DMA_Channel;
   uint32_t       txDMA_Stream;
   uint32_t       rxDMA_Stream;
+  bool           useFIFO;
```

Comment:
`DMA_FIFOMode`. The STM32 hardware specific defs should be consistent with HAL/LL to make it easier.

Reply:
Then it may need a couple of parameters, as there are a few dependent parameters that may need to setup together.

Reply:
Updated, see if it is good now.